docs/standard/serialization/toc.yml

```diff
@@ -67,23 +73,23 @@
       href: add-element-for-schemaimporterextensions.md
     - name: <xmlSerializer> Element
       href: xmlserializer-element.md
-  - name: Serialization Tools
-    href: serialization-tools.md
-    items:
-    - name: XML Serializer Generator Tool (Sgen.exe)
-      href: xml-serializer-generator-tool-sgen-exe.md
-    - name: XML Schema Definition Tool (Xsd.exe)
-      href: xml-schema-definition-tool-xsd-exe.md
-  - name: Serialization Samples
-    href: serialization-samples.md
-    items:
-    - name: Basic Serialization Technology Sample
-      href: basic-serialization-technology-sample.md
-    - name: Custom Serialization Order With XmlSerializer
-      href: custom-serialization-order-with-xmlserializer.md
-    - name: SchemaImporterExtension Technology Sample
-      href: schemaimporterextension-technology-sample.md
-    - name: Version Tolerant Serialization Technology Sample
-      href: version-tolerant-serialization-technology-sample.md
-    - name: Web Services Generics Serialization Technology Sample
-      href: web-services-generics-serialization-technology-sample.md
+    - name: Serialization Tools
```

Comment:
this change in indentation is not 100% accurate. some of the samples are related to binary serialization too. I think it's fine to indent the serialization tools though

Reply:
I'll fix this in a separate PR

* Retire the link farm doc and redirect its URL to the top-level serialization overview doc.
* Put a Samples node under Binary and XML sections
* Divide up the samples into binary and XML, and add nodes under the appropriate Samples nodes for them.

When we have System.Text.Json samples we'll add a Samples node under the JSON section as well.